ui-v2/app/components/feedback-dialog/index.js

```diff
@@ -31,7 +31,7 @@ export default Component.extend(SlotsMixin, {
       .then(() => {
         return new Promise(resolve => {
           this.dom
-            .element(`.${className}`, this.element)
+            .element(`.${className}`, this.$feedback)
             .addEventListener('transitionend', resolve);
```

Comment:
Just a note to self more than anything and if I PR something at point this comment might help you to understand what the PR is about.

I think this component can use the new State component that we have now, and if not we can definitely use `{{on 'transitionend'}}` here so we can add listeners declaratively. Its not something I'd look at immediately but I might link back here for reference if we eventually change it.